docs/user/security/reporting.asciidoc

```diff
@@ -57,6 +57,30 @@ that provides read and write privileges in
 Go to *Management > Users*, add a new user, and assign the user the built-in
 `reporting_user` role and your new custom role, `custom_reporting_user`.
 
+[float]
+==== With a custom index
+
+If you are using Reporting with a custom index,
+start the `xpack.reporting.index` setting
```

Comment:
Maybe:

> If you are using Reporting with a custom index, the `xpack.reporting.index` setting should begin with `.reporting-`. The default {kib} user has `all` privileges against the `.reporting-*` pattern of indices.

The change would make it consistent with the wording where there is similar advice: https://www.elastic.co/guide/en/kibana/current/configuring-reporting.html#report-indices

On that page, it says:
> NOTE: If security is enabled, the `xpack.reporting.index` setting should begin with `.reporting-` in order for the `kibana_system` role to have the necessary privileges over the index.